spa/src/appStore.ts

```diff
@@ -11,6 +11,25 @@ export interface User {
   username?: string,
   token?: string,
   admin?: boolean,
+  hasHome?: boolean,
+}
+
+export interface Place {
```

Comment:
Nothing that I think you need to change now, but it'd be great if we could start sharing types between the API and the client.

Reply:
probably more for a discussion.